demo/src/app/components/+modal/modal-section.list.ts

```diff
@@ -171,6 +172,14 @@ export const demoComponentContent: ContentSection[] = [
           want to defer that until user actually sees modal content. I.e. for a "Select e-mail recipient" modal
           you might want to defer recipient list loading until the modal is shown.</p>`,
         outlet: DemoAutoShownModalComponent
+      },
+      {
+        title: 'Modal window with other components',
+        anchor: 'modal-with-popups',
+        component: require('!!raw-loader?lang=typescript!./demos/modal-with-popups/modal-with-popups.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/modal-with-popups/modal-with-popups.html'),
+        description: `<p>Add other components to modal window (<code>popover</code> and <code>tooltip</code> in this case)</p>`,
```

Comment:
Modals work fine with other components. In this example, there are popover and tooltip in the modal.